src/main/webapp/javascript/user-auth.js

```diff
@@ -0,0 +1,96 @@
+var auth2;
+
+function getId() {
+  return auth2.currentUser.get().getId();
+}
+
+/**
+ * loads the auth2 library for the g-signin API, and then initializes
```

Comment:
Super nit: capitalize the first word in each of the function comment sentence (here and other comments in the file).